docs/scos/dev/tutorials-and-howtos/howtos/howto-delete-store-with-ds.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "HowTo: Delete store when using the dynamic store feature"
+description: This document shows how to delete store when the system is running with a dynamic store feature.
+past_updated: Jun 26, 2023
+template: howto-guide-template
+---
+
+This document describes the steps to consider how manually delete store in spryker when using dynamic store.
+
+## When it might be useful
+
+Often, the store that was created earlier becomes unnecessary. It can be a store that has been used for a brand, product, holiday, or marketing promotions or events аnd ending experiments with no production environment. 
```

Comment:
```suggestion
Occasionally, a store that was created earlier becomes unnecessary. It can be a store that has been used for a brand, product, holiday, marketing promotions, or events аnd ending experiments with no production environment. 
```

Reply:
editor's note:
what does ending experiments with no production environment.  mean exactly in this context? please explain.